docs/reference/ml/ml-shared.asciidoc

```diff
@@ -843,6 +843,33 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-validation-loss-fold]
 For running jobs only, contains messages relating to the selection of a node to 
 run the job.
 
+//Begin data_counts
+`data_counts`:::
+(object)
+An object containing statistical data about the documents in the analysis.
++
+.Properties of `data_counts`
+[%collapsible%open]
+====
+`training_docs_count`:::
+(integer)
+The number of documents that are used for training the model.
+
+`test_docs_count`:::
+(integer)
+The number of documents that are not used for training the model and can be used 
+for testing.
+
+`skipped_docs_count`:::
+(integer)
+The number of documents that are skipped during the analysis because they 
+contained values that are not supported by the analysis. Unsupported values are 
```

Comment:
> Unsupported values are ...

I am getting lost part way through this sentence.  Could we change to something like this?:

> For example, {oldetection} does not support missing fields so it skips documents with missing fields. Likewise, [all types of analysis?] skip documents that contain arrays with more than one element.